lib/util/http.js

```diff
@@ -68,6 +68,24 @@ const xml = contentType('application/xml');
 const atom = contentType('application/atom+xml');
 const json = contentType('application/json');
 
+const binary = (type, name, content) => (_, response) => {
+  response.set('Content-Type', type);
+  response.set('Content-Disposition', `attachment; filename="${name}"`);
```

Comment:
Do we need to sanitize `name`?

Reply:
It looks like this Sentry error is related to `Content-Disposition` containing an invalid character: https://sentry.io/organizations/getodk/issues/2251111354/